code/game/objects/structures/watercloset.dm

```diff
@@ -15,7 +15,21 @@
 	. = ..()
 	open = round(rand(0, 1))
 	update_icon()
-
+	
+/obj/structure/toilet/contraband/Initialize(mapload)
+	..()
+	if(!open && mapload && prob(10))
+		var/ctype = /obj/item/coin/arcade_token
+		if (prob(33))
+			ctype = pick( CONTRABAND_PRISON )
+		else
+			ctype = pick( CONTRABAND_SYNDIE )
+		
+		if (ctype == /obj/item/toy/cards/deck/syndicate)	//hardcoded protection to prevent syndie deck from spawning
+			ctype = /obj/item/coin/arcade_token
```

Comment:
what, why even have it in the loot pool then

Reply:
Because it's the same loot that the Smuggler's Satchel/Contraband Crates use, and I'm not sure if I should remove it from those two as well.